ruby/BUILD.bazel

```diff
@@ -161,6 +162,29 @@ conformance_test(
     }),
 )
 
+genrule(
+    name = "copy_ruby_amalgamation_h",
+    srcs = ["@upb//:ruby-upb.h"],
+    outs = ["generated-in/ext/google/protobuf_c/ruby-upb.h"],
+    cmd = "cp $< $@",
+)
+
+genrule(
+    name = "copy_ruby_amalgamation_c",
+    srcs = ["@upb//:ruby-upb.c"],
+    outs = ["generated-in/ext/google/protobuf_c/ruby-upb.c"],
+    cmd = "cp $< $@",
+)
+
+generated_file_staleness_test(
```

Comment:
If this staleness test fails what are the mitigation steps?  If the upb repo is the source of truth and nobody is editing these directly, this makes sense but might warrant a comment

Reply:
If the staleness test fails it will tell you how to fix it. It will output a message like:

```
To fix run THIS command:
  bazel-bin/cmake/test_generated_files --fix
```

The expectation is that this will (temporarily) fail whenever we update our protobuf->upb dep to point to a newer commit: https://github.com/protocolbuffers/protobuf/blob/f7188186c52b9e3708112a622f47168e3333cf24/protobuf_deps.bzl#L118

Reply:
I guess my point is more that this strategy only works for 1 directional changes.  If changes are coming from both sides this won't work